src/Workspaces/Core/Portable/Formatting/Context/FormattingContext.cs

```diff
@@ -278,12 +280,12 @@ public void AddInitialSuppressOperation(SuppressOperation operation)
                 {
                     // don't bother to calculate line alignment between tokens
                     valuePairs[i] = (operation, shouldSuppress: false, onSameLine: false);
-                    return;
+                    continue;
```

Comment:
i see you mentioned this as an imporant change... but i'm curious why it had to happen in this PR.   Is it fixing a separate bug?

Reply:
in other words, should we not just have a PR that fixes just this with an appropriate test?

I ask because otherwise this PR doesn't seem to be changing behavir, just shuffling things around so tehy can be used from multiple places.  It would be nice to keep that sort of things separate from actual behavioral changes.

Reply:
I inlined a delegate which became a loop.

Reply:
oh ha.  yowza.  what a pain to have to find.